app/database/models/server/channel.ts

```diff
@@ -46,7 +46,7 @@ export default class ChannelModel extends Model implements ChannelModelInterface
         /** A CHANNEL can be associated with multiple CHANNEL_MEMBERSHIP (relationship is 1:N) */
         [CHANNEL_MEMBERSHIP]: {type: 'has_many', foreignKey: 'channel_id'},
 
-        /** A CHANNEL can be associated with multiple CATEGORY_CHANNEL (relationship is 1:N) */
+        /** A CHANNEL can be associated with one CATEGORY_CHANNEL per team (relationship is 1:1) */
```

Comment:
This is a 1:N relationship. You could even argue is a relationship with the CATEGORY table in a N:N relationship.